requirements.txt

```diff
@@ -9,7 +9,7 @@ datadog==0.21.0
 flake8==5.0.4
 Flask==2.2.2
 honcho==1.1.0
-jsonschema==4.16.0
+jsonschema==4.9.1
```

Comment:
Why this change?

Reply:
Good catch, this was from testing out different jsonschema to markdown or restructuredtext libs which required a different version of jsonschema. I've changed this back.